.github/workflows/e2e-tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Set up Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4
```

Comment:
Should we pin this one using a commit SHA as well?

Reply:
According to semgrep not, as it's not considered a 3rd party action.